test/card-test.js

```diff
@@ -143,16 +143,17 @@ describe('cards interactions', function () {
   //   assert.equal('250px', leftPosition, msg2);
   // });
 
-  // it('card dropability can be disabled', function () {
-  //   let card = new Card();
-  //   $(card.card).data('droppable');
-  //   if($(card.card).droppable('disabled')))
-  //     var disabled = 1;
-  //   return assert.equal(disabled, 1);
-  // });
+  it('card dropability can be disabled', function () {
```

Comment:
Similar to my comments on line 124, the `jquery-ui.droppable('disable')` command is a functionality provided by `jquery-ui` package. No need to test `jquery-ui` within our app; that type of testing should be handled by the developers of `jquery-ui` within their package.